Parsers/ASimNetworkSession/ProductParsers/ASimNetworkSessionPaloAltoPanOS.yaml

```diff
@@ -0,0 +1,118 @@
+Parser:
+  Title: Palo Alto Network Sessions
+  Version: '0.1'
+  LastUpdated: Dec 20, 2021
+Product:
+  Name: PaloAlto
+Normalization:
+  Schema: NetworkSessions
+  Version: '0.2.0'
+References:
+- Title: ASIM Network Session Schema
+  Link: https://aka.ms/ASimNetworkSessionDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+Description: |
+  This Query Parser maps Palo Alto PanOS Firewall Events (CommonSecurityLogs) to the Advanced SIEM Information Model Network Session schema.
+ParserName: ASimNetworkSessionPaloAltoPanOS
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let NWParser=(disabled:bool=false){
+  CommonSecurityLog | where not(disabled)
+  | where DeviceVendor=="Palo Alto Networks" and (Activity=="TRAFFIC")
+  | parse AdditionalExtensions with "PanOSDstPackets="DstPackets:long";PanOSsrc_packets="SrcPackets:long";start="EventStartTime:datetime";reason="*
+  | project-rename 
+        EventVendor=DeviceVendor // Not documented
+      , EventProduct=DeviceProduct // Not Documented
+      , EventProductVersion=DeviceVersion // Not Documented
+      , DvcHostname=DeviceName   
+      , EventResourceId=_ResourceId  // Not Documented 
+      , NetworkApplicationProtocol=ApplicationProtocol
+      , SrcZone=DeviceCustomString4 
+      , DstZone=DeviceCustomString5 
+      , NetworkRuleName=DeviceCustomString1
+      , NetworkProtocol=Protocol
```

Comment:
Do we know if correct capitalization is used?

Reply:
From PA docs, it should be CAPITALIZED

Reply:
I checked in online samples and the value is lowercase and needs to be uppercased.